mmv1/products/workbench/Instance.yaml

```diff
@@ -0,0 +1,470 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+base_url: "projects/{{project}}/locations/{{location}}/instances"
+create_url: "projects/{{project}}/locations/{{location}}/instances?instanceId={{name}}"
+self_link: "projects/{{project}}/locations/{{location}}/instances/{{name}}"
+id_format: "projects/{{project}}/locations/{{location}}/instances/{{name}}"
+import_format:
+  - "projects/{{project}}/locations/{{location}}/instances/{{name}}"
+name: Instance
+description: A Workbench instance.
+update_verb: :PATCH
+update_mask: true
+iam_policy: !ruby/object:Api::Resource::IamPolicy
+  method_name_separator: ":"
+  fetch_iam_policy_verb: :GET
+  parent_resource_attribute: "instance_name"
+  import_format:
+    [
+      'projects/{{project}}/locations/{{location}}/instances/{{instance_name}}',
```

Comment:
instance_name is not a field that exists on this resource, I think you mean {{name}}?

Reply:
Were you able to successfully run the generated IAM tests on this resource?

Reply:
Is there documentation on how the iam tests are generated and how they are tun?